### PR TITLE
wip: Bump polkadot to v0.9.13

### PR DESCRIPTION
**Pull request by NunoAlexandre**
- [x] Bump version of polkadot
- [x] Bump substrate
- [x] Bump cumulus
- [x] Fix compilation issues
- [x] Run tests
- [x] Update docker image
- [x] Ensure that the relay-chain produces blocks
- [ ] Ensue that the para-chain produces blocks

### CODE REVIEW

**Comment by NunoAlexandre** [comment]
`State dump`

As of #https://github.com/centrifuge/centrifuge-chain/pull/527/commits/a62577a8976cd16cb5ae32573a1dccbf63de7ae7, running `cargo build` fails with the following error:

```
   Compiling beefy-gadget-rpc v4.0.0-dev (https://github.com/paritytech/substrate?branch=polkadot-v0.9.13#de6033a2)
error[E0308]: mismatched types
  --> /Users/n/.cargo/git/checkouts/polkadot-4038f27d5e4ea2e8/bff00af/node/network/protocol/src/authority_discovery.rs:51:3
   |
51 |         AuthorityDiscoveryService::get_addresses_by_authority_id(self, authority).await
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `Vec`, found struct `HashSet`
   |
   = note: expected enum `Option<Vec<Multiaddr>>`
              found enum `Option<HashSet<Multiaddr>>`
note: return type inferred to be `Option<Vec<Multiaddr>>` here
```

**Comment by NunoAlexandre** [comment]
@mustermeiszer what is the best base branch for this bump? 🤔

**Comment by mustermeiszer** [comment]
Parachain. We rebase rt3-rc2 and rt4 then. Or rebase this one, when we device to not bring in polkadot-v0.9.13 in our side. Its a bit of hustle to test, if there where any migrations or other changes on the substrate side. But we will see.

**Comment by NunoAlexandre** [comment]
> Parachain. We rebase rt3-rc2 and rt4 then. Or rebase this one, when we device to not bring in polkadot-v0.9.13 in our side. Its a bit of hustle to test, if there where any migrations or other changes on the substrate side. But we will see.

Cool, thanks man!

File: docker-compose-local-relay.yml [hunk 1]
```diff
@@ -13,24 +13,14 @@ services:
...
-      --rpc-port 9933
-      --ws-port 9944
-      --rpc-external
-      --rpc-cors all
-      --ws-external
-      --rpc-methods=Unsafe
-      --alice
+      --chain=/chainspec.json --wasm-execution=compiled --execution=wasm
+      --no-beefy --base-path=/data --port 30333 --rpc-port 9933 --ws-port 9944
+      --rpc-external --rpc-cors all --ws-external --rpc-methods=Unsafe --alice
```
Comment (NunoAlexandre) [inline docker-compose-local-relay.yml:18]: These changes must have been caused by my editor, not intentional 🙃

**Comment by NunoAlexandre** [comment]
Closed in favour of https://github.com/centrifuge/centrifuge-chain/pull/611

**Pull request closed without merging**